### DISCUSSION TITLE
skeleton docker with maintenance cron job

### DISCUSSION THREAD

**Discussion by leo-schick** [pimcore/pimcore#11952] [Q&A / Support] [answered]
I use the [pimcore/skeleton](https://github.com/pimcore/skeleton) repo as basis for my project and just wonder why the default docker-compose setup does not include the maintenance cron job defined [here](https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html#page_5-Maintenance-Cron-Job).

Is there any specific reason for this?

Ias I understand it, this maintenance job should always be part of one environment. Or am I wrong here?

**Comment by dvesh3** [comment, marked as answer]
Hi, @leo-schick starting with pimcore version 10.2, maintenance tasks are handled by Symfony Messenger and skeleton docker image comes with [supervisor worker](https://github.com/pimcore/skeleton/blob/10.x/docker-compose.yaml#L52-L59) which is responsible for executing maintenance tasks.

Please see more information here https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/Upgrade_Notes/index.html#page_10-2-0

**Reply by leo-schick** [reply]
@dvesh3 Good to know!

I suggest then to update the documentation here:
https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html#page_5-Maintenance-Cron-Job

**Reply by dvesh3** [reply]
done https://github.com/pimcore/pimcore/commit/f89a4990fb0804e439c611399ed774e3744ce48b
